modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/test-ddm-structure-definition-with-nested-fields.json

```diff
@@ -0,0 +1,450 @@
+{
+	"availableLanguageIds": [
+		"en_US"
+	],
+	"defaultLanguageId": "en_US",
+	"definitionSchemaVersion": "2.0",
+	"fields": [
+		{
+			"collapsible": false,
+			"ddmStructureId": "",
+			"ddmStructureKey": "",
+			"ddmStructureLayoutId": "",
+			"fieldReference": "Fieldset1",
+			"label": {
+				"en_US": "Fields Group 1"
+			},
+			"labelAtStructureLevel": true,
+			"localizable": false,
+			"name": "Fieldset1",
+			"nestedFields": [
+				{
+					"confirmationErrorMessage": {
+						"en_US": ""
+					},
+					"confirmationLabel": {
+						"en_US": ""
+					},
+					"dataType": "string",
+					"direction": [
+						"vertical"
+					],
+					"displayStyle": "singleline",
+					"fieldNamespace": "",
+					"fieldReference": "Text1",
+					"hideField": false,
+					"indexType": "keyword",
+					"label": {
+						"en_US": "1"
```

Comment:
Can you edit it to "Text 1," please?🥹
Do this to the other's Text Label.

Reply:
Of course! Done. Thx!!